example-wasm/src/atlas-app-services/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ATLAS_APP_ID": "application-0-pbvdy"
```

Comment:
Is it safe to leave the app id in the open?

Reply:
I don't believe the App ID is a secret, but will definitely replace this with a placeholder 👍

Reply:
Done ✅